webdriver-spec.html

```diff
@@ -1435,288 +1435,532 @@
 <section>
 <h2>Capabilities</h2>
 
-<p>WebDriver <dfn>capabilities</dfn> allow the <a>local end</a>
- to specify what features it desires or requires the <a>remote end</a> to fulfill
- to be able to create a <a>new session</a>.
-
-<p>The following table provides an overview
- of recognised capabilities, and their usage and effect:
+<p>WebDriver <dfn>capabilities</dfn> are used to communicate the features
+supported by a given implementation. The <a>local end</a> may use capabilities
+to define which features it requires the <a>remote end</a> to satisfy when
+creating a <a>new session</a>. Likewise, the <a>remote end</a> uses capabilities
+to describe the full feature set for a session.
+
+<p>The following table provides an overview of the standard capabilities that
+each implementation <em>MUST</em> support. An implementation <em>MAY</em> define
+additional capabilities, although it is <em>RECOMMENDED</em> that custom
+capabilities prepend the capability key with a vendor prefix, as outlined in
+[[CSS21]].
```

Comment:
The section on protocol extension talks about vendor prefixs used in the URI path. There's no mention of extending parameters. Should that be added?